src/Serilog/Context/ImmutableStack.cs

```diff
@@ -68,9 +78,11 @@ System.Collections.IEnumerator System.Collections.IEnumerable.GetEnumerator()
 
         public T Top => _top;
 
+#if REMOTING
         void ISerializable.GetObjectData(SerializationInfo info, StreamingContext context)
```

Comment:
Perhaps we could move this up into the first `#if REMOTING` block to piggy-back on its `#if`?